File: TravelGuide/app/src/main/java/com/travel/travelguide/activity/IntroActivity.java
package com.travel.travelguide.activity;

import android.content.Intent;
import android.os.Bundle;

import com.travel.travelguide.databinding.ActivityIntroBinding;

public class IntroActivity extends BaseActivity {

    ActivityIntroBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityIntroBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.btnIntro.setOnClickListener(v -> startActivity(new Intent(IntroActivity.this, MainActivity.class)));
    }
}

File: HospitalAppointment/app/src/main/java/com/appointment/hospitalappointment/Activity/IntroActivity.kt
package com.appointment.hospitalappointment.Activity

import android.content.Intent
import android.os.Bundle
import com.appointment.hospitalappointment.databinding.ActivityIntroBinding

class IntroActivity : BaseActivity() {

    private lateinit var binding: ActivityIntroBinding

    override fun onCreate(savedInstanceState: Bundle?) {

//        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN)

        super.onCreate(savedInstanceState)
        binding = ActivityIntroBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.apply {
            btnStart.setOnClickListener{
                startActivity(Intent(this@IntroActivity, MainActivity::class.java))
            }
        }
    }
}